### PR TITLE
API-7560 DatamartPractitionerRole missing fields

### PR DESCRIPTION
**Pull request by jblefkove-va**

### CODE REVIEW

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java [hunk 1]
```diff
@@ -158,4 +170,37 @@
...
+    public enum System {
+      phone,
+      fax,
+      pager,
+      email
+    }
+
+    public enum Use {
+      home,
+      mobile
```
Comment (nmoisan) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java:203]: also needs to include `work` as a possible value. I think I left that one out of the JSON documentation.

**Comment by nmoisan** [comment]
LGTM other than the one comment on `use` enum

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java [hunk 2]
```diff
@@ -25,6 +25,8 @@
 
   private Optional<DatamartReference> practitioner;
 
+  private boolean active;
```
Comment (apeirats-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java:28]: `Boolean`?

Comment (jblefkove-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java:28]: This field is guaranteed non-null

**Review by jhulbert-va — APPROVED** [review]

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java (hunk 2 above)
Comment (apeirats-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java:28]: Will need to add them to synth dm

Comment (jblefkove-va) [reply data-query/src/main/java/gov/va/api/health/dataquery/service/controller/practitionerrole/DatamartPractitionerRole.java:28]: Good point, will go with `Boolean` for now then

**Review by apeirats-va — APPROVED** [review]

**Review by eclendenning-va — APPROVED** [review]

**Pull request merged**